<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>thermal_factor_editor</class>
 <widget class="QDialog" name="thermal_factor_editor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>548</width>
    <height>276</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Edit Thermal Factor</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QCheckBox" name="is_tensor">
     <property name="text">
      <string>Tensor</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTabWidget" name="tab_widget">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="scalar_tab">
      <attribute name="title">
       <string>Scalar</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="ScientificDoubleSpinBox" name="scalar_value">
         <property name="decimals">
          <number>8</number>
         </property>
         <property name="minimum">
          <double>-1000000.000000000000000</double>
         </property>
         <property name="maximum">
          <double>10000000.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tensor_tab">
      <attribute name="title">
       <string>Tensor</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <layout class="QVBoxLayout" name="tensor_editor_layout"/>
       </item>
       <item row="1" column="0">
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="button_box">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ScientificDoubleSpinBox</class>
   <extends>QDoubleSpinBox</extends>
   <header>scientificspinbox.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>is_tensor</tabstop>
  <tabstop>tab_widget</tabstop>
  <tabstop>scalar_value</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>button_box</sender>
   <signal>accepted()</signal>
   <receiver>thermal_factor_editor</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>273</x>
     <y>565</y>
    </hint>
    <hint type="destinationlabel">
     <x>273</x>
     <y>295</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>button_box</sender>
   <signal>rejected()</signal>
   <receiver>thermal_factor_editor</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>273</x>
     <y>565</y>
    </hint>
    <hint type="destinationlabel">
     <x>273</x>
     <y>295</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
